packages/suite/src/components/wallet/TransactionItem/TransactionItemBlurWrapper.tsx

```diff
@@ -1,6 +1,7 @@
 import styled, { css } from 'styled-components';
 
 export const BlurWrapper = styled.span<{ $isBlurred: boolean }>`
+    width: 100%;
```

Comment:
does it do something with non-block element like `span`?

Reply:
Yes, I added that

Reply:
What does it do?

Reply:
now seems like it does nothing, not sure why I added it before as well as width, seems to be ok without it so i removed it